app/templates/resources/update_review.html

```diff
@@ -0,0 +1,54 @@
+{% extends 'layouts/base.html' %}
+{% import 'macros/form_macros.html' as f %}
+
+{% block content %}
+    <div class="ui stackable centered grid container">
+        <div class="twelve wide column">
+            <table class="ui compact definition table">
+                <tr><td>Name</td><td>{{ resource.name }}</td></tr>
+                <tr><td>Description</td><td>{{ resource.description }}</td></tr>
+                <tr><td>Website</td><td>{{ resource.website }}</td></tr>
+                <tr><td>Address</td><td>{{ address.street_address }}</td></tr>
+                <tr><td>User</td><td>{{ user.full_name() }}</td></tr>
+                <tr><td>Reviews</td><td>
```

Comment:
```html
<tr>
    <td>Reviews</td>
    <td>
        {% for review in resource.reviews.all() %}
            {{ review.content }}
        {% endfor %}
    </td>
</tr>
```
^^ new formatting and also let's pass in the reviews variable with all of the reviews inside it instead of querying here